WordPress/WordPressApi/WordPressComRestApi.swift

```diff
@@ -59,6 +61,24 @@ public class WordPressComRestApi: NSObject
         return sessionManager
     }()
 
+    private var uploadSession: NSURLSession {
+        get {
```

Comment:
No need to specify the `get` block, if (all the computed var can do) is return a new entity